lib/testing/src/lib/core/actions/users.actions.ts

```diff
@@ -22,7 +22,7 @@ import * as remote from 'selenium-webdriver/remote';
 import { browser } from 'protractor';
 import { ImageUploadRepresentation, UserRepresentation } from '@alfresco/js-api';
 import { ApiService, IdentityService, UserModel, Logger } from '@alfresco/adf-testing';
-import { Tenant } from '../models/APS/tenant';
+import { Tenant } from './tenant';
 
 export class UsersActions {
```

Comment:
I think this class needs some refactoring before being moved to adf testing package. First of all, the createUser method it's not working in case you want to have a user created on ACS + APS1 + Identity, because the admin users are different: for keycloak the admin user that can create user is different from the admin users on ACS and APS1. Besides that, i would say it's not so easy to understand that method, i would split it and refactor it taking in consideration the clean coding practices.